plugins/httpapi/dcnm.py

```diff
@@ -62,6 +62,12 @@ def get_version(self):
     def set_version(self, version):
         self.version = version
 
+    def set_token(self, token):
+        self.token = token
```

Comment:
Why is this needed?  Does this have any impact on other modules in the collection?  Did we run tests to confirm?

Reply:
Please add a comment for why this is needed as discussed during our call

Reply:
Sure, will do